### PR TITLE
Import cluster with flows

### PR DESCRIPTION
**Pull request by anivargi**

### CODE REVIEW

**Comment by anivargi** [comment]
@brainfunked please review.

File: lib/tendrl/flow.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
+module Tendrl
+  class Flow
+
+    METHOD_MAPPING = { 'create' => 'POST', 'update' => 'PUT', 'delete' =>
```
Comment (brainfunked) [inline lib/tendrl/flow.rb]: While such convention based approach is fine, it needs to be standardised and made part of the common library across the project.

File: base.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+require 'bundler'
+require 'yaml'
+require 'sinatra/base'
+require "sinatra/multi_route"
+require 'securerandom'
+require 'active_support/core_ext/hash'
+require 'active_support/inflector'
+require './lib/tendrl'
```
Comment (brainfunked) [inline base.rb]: It shouldn't be necessary to use relative paths to the lib directory for `require`. It must be possible to add the `lib` directory to the ruby library path environment variable at startup, via config.ru or equivalent. I've previously done this in the unicorn startup file. If puma and shotgun are capable of evaluating config.ru, then it should be added there.

File: cluster.rb [hunk 3]
```diff
@@ -72,11 +34,11 @@ class App < Sinatra::Base
     job_id = SecureRandom.uuid
     etcd.set("/queue/#{job_id}", value: {
       cluster_id: params[:cluster_id],
-      sds_nvr: cluster['sds_version'],
-      action: params[:action],
-      object_type: params[:object_type],
-      status: 'processing',
-      attributes: body.slice(*component.attributes.keys)
+        sds_nvr: cluster['sds_version'],
```
Comment (brainfunked) [inline cluster.rb]: The extra indents here seem to be unnecessary?

File: base.rb (hunk 2 above)
Comment (anivargi) [inline base.rb]: Initially had used just `require 'tendrl'` assuming the server loads the lib folder automatically, but that needed me to add -Ilib everytime, so just added the require relative instead.

File: cluster.rb (hunk 3 above)
Comment (anivargi) [inline cluster.rb]: The file is going to be revisited once we have the new yaml file for clusters

File: base.rb (hunk 2 above)
Comment (brainfunked) [inline base.rb]: The lib folder will be loaded only if it's in the ruby load path, which happens during the initialization. It's just a push to the environment variable. I don't know whether shotgun and puma load config.ru, but if they do, just add that line to config.ru and it'll work.

**Pull request merged**